### PR TITLE
[OBSDEF - 9509] - word-break for AlertItem component

### PR DESCRIPTION
**Pull request by GauravMenezes-Professional**
-> Updated package.json, added word-break to Alert Text, and performed yarn-test while using this as dependency to make sure no tests fail.

### CODE REVIEW

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM

File: src/emphasis/alert/AlertItem.tsx [hunk 1]
```diff
@@ -8,58 +8,50 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (maheshkulkarni4) [inline src/emphasis/alert/AlertItem.tsx:8]: Add Storybook usecase to test his scenario.

Comment (GauravMenezes-Professional) [reply src/emphasis/alert/AlertItem.tsx:8]: Added storybook usecase for this scenario.

File: src/emphasis/alert/AlertItem.tsx [hunk 2]
```diff
@@ -8,58 +8,50 @@
...
-    actions?: React.ReactElement
-    type?: AlertType
-    children?: React.ReactNode | React.ReactNode[]
-    icon?: React.ReactElement
-    static?: boolean
+    actions?: React.ReactElement;
+    type?: AlertType;
+    children?: React.ReactNode | React.ReactNode[];
+    icon?: React.ReactElement;
+    static?: boolean;
```
Comment (akate1) [inline src/emphasis/alert/AlertItem.tsx:22]: Please add some documentation for this props

File: src/emphasis/alert/AlertItem.tsx [hunk 3]
```diff
@@ -8,58 +8,50 @@
...
             });
         });
     }
 
     render() {
         const {actions, type, children, icon} = this.props;
         let classNames = ["alert-item"];
-        if (this.props.static)
-            classNames.push("static");
+        if (this.props.static) classNames.push("static");
```
Comment (akate1) [inline src/emphasis/alert/AlertItem.tsx:47]: extract `static `at line 45

File: src/emphasis/alert/AlertItem.tsx [hunk 4]
```diff
@@ -8,58 +8,50 @@
...
-                    {AlertItem.iconWithAlertClass(type, icon)}
-                </div>
+                <div className="alert-icon-wrapper">{AlertItem.iconWithAlertClass(type, icon)}</div>
                 <div className="alert-text">
-                    {children}
-                </div>
-                {actions &&
-                <div className="alert-actions">
-                    {AlertItem.renderActions(actions)}
+                    <div style={{wordBreak: "break-word"}}>{children}</div>
```
Comment (akate1) [inline src/emphasis/alert/AlertItem.tsx:52]: Please create constant for this style

**Review by akate1 — APPROVED** [review]
LGTM

File: src/emphasis/alert/AlertItem.tsx (hunk 3 above)
Comment (GauravMenezes-Professional) [reply src/emphasis/alert/AlertItem.tsx:47]: As static is a keyword in typescript it is giving error if extracted hence taking it from props

File: src/emphasis/alert/AlertItem.tsx (hunk 2 above)
Comment (GauravMenezes-Professional) [reply src/emphasis/alert/AlertItem.tsx]: Documentation done

File: src/emphasis/alert/AlertItem.tsx (hunk 3 above)
Comment (GauravMenezes-Professional) [reply src/emphasis/alert/AlertItem.tsx]: Changed prop to isStatic and changed it in story as well

File: src/emphasis/alert/AlertItem.tsx (hunk 4 above)
Comment (GauravMenezes-Professional) [reply src/emphasis/alert/AlertItem.tsx]: Constant created

**Pull request merged**